trendyqc/trend_monitoring/models/bam_qc.py

```
@@ -145,6 +145,9 @@ class Picard(models.Model):
         blank=True,
         null=True,
     )
+    rnaseq_metrics = models.ForeignKey(
+        "RNAseq_metrics", on_delete=models.DO_NOTHING, blank=True, null=True
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)